docs/release-notes/.FSharp.Compiler.Service/8.0.200.md

```diff
@@ -13,3 +13,4 @@
 * Parser recovers on unfinished enum case declarations. ([PR #16401](https://github.com/dotnet/fsharp/pull/16401))
 * Parser recovers on unfinished record declarations. ([PR #16357](https://github.com/dotnet/fsharp/pull/16357))
 * `MutableKeyword` to [SynFieldTrivia](../reference/fsharp-compiler-syntaxtrivia-synfieldtrivia.html) ([PR #16357](https://github.com/dotnet/fsharp/pull/16357))
+* Parser recovers on complex primary constructor patterns, better tree representation for primary constructor patterns ([PR #16425](https://github.com/dotnet/fsharp/pull/16425))
```

Comment:
Nitpick but this should also have a `### Changed` entry in my book.
FCS consumers need to be made aware that `implicitCtorSynPats` went from `SynSimplePats` to `SynPat`. This affects a lot of people and should be pointed out as such.

Reply:
@nojaf Would you like to send a PR to share an example of what you'd like to see here?

Reply:
```suggestion
* Parser recovers on complex primary constructor patterns, better tree representation for primary constructor patterns. ([PR #16425](https://github.com/dotnet/fsharp/pull/16425))

### Changed
* `implicitCtorSynPats` in `SynTypeDefnSimpleRepr.General` is now `SynPat option` instead of `SynSimplePats option`. ([PR #16425](https://github.com/dotnet/fsharp/pull/16425))
*  `SyntaxVisitorBase<'T>.VisitSimplePats` now takes `SynPat` instead of `SynSimplePat list`. ([PR #16425](https://github.com/dotnet/fsharp/pull/16425))
```